src/components/ArticleCard.jsx

```diff
@@ -20,35 +21,33 @@ const ArticleCard = (props) => {
     },[])
 
     if (isLoading) {
-        return <p>Loading...</p>
+        return <p></p>
     }
     
     if (isError) {
         return <p>Sorry, article not found</p>
     }
 
     const shortDate = article.created_at.slice(8,10)+"/"+article.created_at.slice(5,7)+"/"+article.created_at.slice(0,4)
-    const formattedDate = `${shortDate} ${article.created_at.slice(12,16)}`
-    
+    const formattedDate = `${shortDate} at ${article.created_at.slice(11,16)}`
+
     return (
         <section id="article-card">
             <ul className="display-line1">
```

Comment:
You are using a `ul` here but with not `li` tags inside, perhaps this should be a different tag